code/script.js

```diff
@@ -229,97 +264,145 @@ const setSecret = () => {
 
 // This function to start (and restart) the game
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
+  totalSeconds = 0
+  valString = ""
+  secondsLabel.innerHTML = ""
+  minutesLabel.innerHTML = ""
+  countGuess.innerHTML = "Guesses: "
```

Comment:
You can have the code in line 396 here  countGuess.innerHTML = "Guesses:" + count 
then let count = 0 so you would be able to see this when the page loads